app/models/user.rb

```diff
@@ -0,0 +1,12 @@
+class User < ApplicationRecord
+  validates :name, length: { minimum: 1 }
```

Comment:
- [ ] It will be great to validate the presence of the name field. This is a requirement for this project. Example: ` validates :name, presence: true`